src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/ComparisonController.kt

```diff
@@ -147,6 +150,58 @@ class ComparisonController(
     return comparisonService.getComparisonPersonByShortReference(comparisonReference, mismatchReference)
   }
 
+  @PostMapping(value = ["{comparisonReference}/mismatch/{mismatchReference}/discrepancy"])
+  @PreAuthorize("hasAnyRole('ROLE_RELEASE_DATE_COMPARER')")
```

Comment:
Should this include `ROLE_RELEASE_DATE_MANUAL_COMPARER`?

Reply:
Yes I've added the manual comparison version of these endpoints and added the role there.